.travis.yml

```diff
@@ -5,9 +5,23 @@ php:
   - 5.5
   - 5.6
   - 7.0
+  - 7.1
   - hhvm
 
-install: ./travis-init.sh
+sudo: false
+
+addons:
+  apt:
+    packages:
+      - libevent-dev # Used by 'event' and 'libevent' PHP extensions
+
+cache:
+  directories:
+    - $HOME/.composer/cache
```

Comment:
Changing this to `$HOME/.composer/cache/files` will ensure we only cache package files and not the meta files.